File: 0171/0171.c
// Author: washing
// DateTime: 2021/1/27 10:33
// File: 0171.c
// Desc: 

int titleToNumber(char * s){
    int n=0;
    int idx = 0;
    while (idx < strlen(s)){
        n = n*26 + ((int)s[idx] - 64);
        idx++;
    }
    return n;
}

File: 0171/0171.js
// Author: washing
// DateTime: 2021/7/30 11:16
// File: 0171.js
// Desc: 


/**
 * @param {string} columnTitle
 * @return {number}
 */
var titleToNumber = function(columnTitle) {
    var n = 0;
    var i;
    for (var idx=0; idx < columnTitle.length; idx++){
        i = columnTitle[idx];
        n = n*26 + i.charCodeAt() - 64;
    }
    return n;
};